ocaml/libs/clock/date.ml

```diff
@@ -150,6 +150,8 @@ let strip_tz tz t =
   in
   {t; tz= None}
 
+let to_utc {t; _} = {t; tz= utc}
```

Comment:
As per the design, the value should be forbidden from being converted if the input tz is None. It should either return a `Date.t option` or raise an exception. I prefer the first and handle it at the caller site.